glossary.md

```diff
@@ -20,9 +23,16 @@ Device
 Director
     A Component which takes part in orchestrating a (i.e. LECO-controlled) measurement setup, see {ref}`components.md#director`.
 
+Directory
+    Each Coordinator maintains a local Directory with all the Components connected to it, and a global Directory with all Components in the Network, see {ref}`control_protocol.md#directory`.
```

Comment:
I don't know, whether it is necessary to indicate how the Coordinator maintains the Directory.
Your change seems to imply, that the Directories differ in the type of names stored, but the difference is the scope: Components of the Node and Components of the Network.

The Coordinator can store the local Directory with Full names as well, if it wishes.
The global Directory has to have full names.

What about:
```suggestion
    Each Coordinator maintains a local Directory with all the Components connected to it (i.e. Components of the Node), and a global Directory with all Components in the whole Network, see {ref}`control_protocol.md#directory`.
```

Reply:
I agree with Benedikt. We have to take care to specify (observable) behaviour, and not implementation details (it's hard to separate at times). Also, we need to define the (technical) terms we use, so that we can understand  (here local vs. global directory) and use them efficiently in the spec.

Reply:
Benedikt, FYI, that there is a slight mismatch between your proposed definition of local directory, and what the main text says, i.e. 
> Each Coordinator shall have a list of the Components (including other Coordinators) connected to it.
This is its _local Directory_.

Other Coordinators are not "of the Node".

Reply:
I'm not sure though why we need separate global and local directories? Probably I have not picked up on all the conversation. 
Why don't we have one directory with every Coordinator? If an entry is local or global will be immediately clear from the Node namespace, and can be filtered as such at will.

Reply:
We should separate them at least in technical terms, as the local Directory is necessary for routing (it contains the connection identities as well).
And a Coordinator is the primary source of truth of that Directory.

The global directory of a Coordinator (which could be defined as "its copy of the union of all local Directories") is an aid for the Components in the Network.

If a Coordinator maintains only one Directory or a local version and a global one, is its one problem.

Reply:
Let us discuss it in #46 , as it is related.

Reply:
I updated the description a little bit.

Reply:
Thanks, sounds good. Let's resolve this and feed back in the result of #46 later.